lib/client-assets.php

```diff
@@ -133,7 +133,7 @@ function wc_admin_register_script() {
  * @param array $statuses Order statuses.
  * @return array formatted statuses.
  */
-function format_order_statuses( $statuses ) {
```

Comment:
Looks like there are a few more spots where `format_order_statuses()` is being used:

```
lib/admin.php
523:		'options'     => format_order_statuses( wc_get_order_statuses() ),
532:		'options'     => format_order_statuses( wc_get_order_statuses() ),
```

Reply:
Hi Timmy,

Thank you for reviewing my PR and sharing your feedback. I have implemented the changes and updated my PR. Please review it and share your feedback on it.

Thanks & Regards,
Ronak